doc/api/crypto.md

```diff
@@ -1683,6 +1810,11 @@ signing algorithms. Optional `options` argument controls the
 ### crypto.generateKeyPair(type, options, callback)
 <!-- YAML
 added: v10.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/24234
+    description: The `generateKeyPair` and `generateKeyPairSync` functions now
+                 produce key objects if no encoding was specified.
```

Comment:
is that semver-major? I think it might be. The current defaults don't seem to be documented, am I missing them? Would be worth working hard to avoid a semver-major change, I'd like to see this API in 11.x. If necessary, you could introduce an pseudo-encoding of `'key'` -- kindof how many of the crypto APIs support (but don't document) that an encoding of `'buffer'` means return a Buffer. Since no one is currently specifying an encoding of `'key'`, this would become backwards compat. In the future, the default can change in a semver-major, if there is appetite for that.

Reply:
I don't think this is semver-major. Currently, the `generateKeyPair` API throws if `publicKeyEncoding` / `privateKeyEncoding` is undefined, and this change makes it succeed instead.